packages/cactus-common/src/main/typescript/public-api.ts

```diff
@@ -3,3 +3,11 @@ export { Logger, ILoggerOptions } from "./logging/logger";
 export { LogLevelDesc } from "loglevel";
 export { Objects } from "./objects";
 export { Strings } from "./strings";
+export {
+  JsObjectSigner,
+  IJsObjectSignerOptions,
+  SignatureFunction,
+  VerifySignatureFunction,
+  HashFunction,
+} from "./js-object-signer";
+export { ISignerKeyPairs } from "./signer-key-pairs";
```

Comment:
Also need to export the class that's in there and rename the file itself to resemble the class name which delivers the main functionality (the interface is just an auxiliary type attached to the class so that's why I recommend naming the file after the class not the interface). As an alternative it also works great if you de-couple the two types into their own files respectively.

Reply:
Refactor the class and update public-api.ts.